README.md

```diff
@@ -85,3 +85,16 @@ Settings:
 * Weight: 100 (min)
 * Grade: 0 (middle)
 * Optical Size: 48 (max)
+
+### Testing
+
+We test the contents of our docs via [`testbook`](https://github.com/nteract/testbook).
+These tests stand up real infrastructure by executing cells in the documentation
+and expect a GCP credential to live at `~/.gcp/doc-test-creds.json`. We'll migrate
+to workload identity pools using github as supporting CI is created.
+
+Tests can be run via:
+
+```bash
+make test
```

Comment:
Perhaps call out that this should be called while in the devcontainer?

Reply:
This actually all works outside of a devcontainer.

I abandoned the devcontainer after docker-in-docker and docker-from-docker woes. I really only kept the devcontainer bits around because the dockerfile will be useful to convert to a setup script when this lands in github actions. I'll likely all devcontainer bits if I poke at it again and can't figure it out.